cfme/tests/candu/test_graph_groupbytag.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime
+from datetime import timedelta
+
+import pytest
+
+from cfme.common.candu_views import UtilizationZoomView
+from cfme.tests.candu import compare_data_with_unit
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.log import logger
+
+
+pytestmark = pytest.mark.uncollectif(lambda appliance: appliance.is_pod)
+
+HOST_GRAPHS = ['host_cpu',
+               'host_memory',
+               'host_disk',
+               'host_network',
+               'host_cpu_state']
+
+INTERVAL = ['Hourly', 'Daily']
+
+GROUP_BY = ['VM Location']
+
+CANDU_VM = 'cu-24x7'
+
+
+@pytest.fixture(scope='function')
+def host(temp_appliance_extended_db):
+    vm = temp_appliance_extended_db.rest_api.collections.vms.get(name=CANDU_VM)
+
+    vm_host = vm.host.name
+    return temp_appliance_extended_db.collections.hosts.instantiate(name=vm_host)
+
+
+@pytest.mark.parametrize('gp_by', GROUP_BY, ids=['vm_tag'])
+@pytest.mark.parametrize('interval', INTERVAL)
+@pytest.mark.parametrize('graph_type', HOST_GRAPHS)
+def test_tagwise(candu_db_restore, interval, graph_type, gp_by, host):
+    """Tests for grouping host graphs by VM tag for hourly and Daily intervals
+
+    prerequisites:
+        * DB from an appliance on which C&U is enabled
+        * DB should have C&U data collection enabled for Tag category
+        * DB should have a VM tagged with proper tag category
+
+    Steps:
+        * Navigate to Host Utilization Page
+        * Select interval(Hourly or Daily)
+        * Select group by option with VM tag
+        * Check graph displayed or not
+        * Zoom graph to get Table
+        * Check tag assigned to VM available in chart legends
+        * Compare table and graph data
+
+    Bugzillas:
+        * 1367560
+
+    Polarion:
+        assignee: nachandr
+        initialEstimate: 1/4h
```

Comment:
@nachandr can you please add `casecomponent`? also without `Tire` we unable to decide `caselevel`. so please add docstring changes.